src/json/ProgramTransaction.schema.json

```diff
@@ -292,13 +292,19 @@
             },
             "analyticsKey": {
               "type": "string",
-              "description": "The type of analytic recorded (i.e. in program goals this could describe the goal achieved such as 'friendPaid')",
-              "pattern": "^[a-zA-Z0-9_]*$"
+              "description": "The type of analytic recorded (i.e. in program goals this could describe the goal achieved such as 'friendPaid'). It maps directly to the objectiveId.",
```

Comment:
```suggestion
              "description": "The type of analytic recorded (i.e. in program goals this could describe the goal achieved such as 'friendPaid'). It maps directly to the goalId.",
```
```suggestion
              "description": "The type of analytic recorded (i.e. in program goals this could describe the goal achieved such as 'friendPaid'). It maps directly to the objectiveId.",
```